pkg/otlp/internal/serializerexporter/consumer.go

```diff
@@ -21,14 +21,26 @@ import (
 var _ translator.Consumer = (*serializerConsumer)(nil)
 
 type serializerConsumer struct {
-	series   metrics.Series
-	sketches metrics.SketchSeriesList
+	cardinality string
+	series      metrics.Series
+	sketches    metrics.SketchSeriesList
+}
+
+// enrichedTags of a given dimension.
+// In the OTLP pipeline, 'contexts' are kept within the translator, and,
+// therefore, this works a little differently than for DogStatsD/check metrics.
+func (c *serializerConsumer) enrichedTags(dimensions *translator.Dimensions) []string {
+	tb := tagset.NewHashlessTagsAccumulator()
+	// Append to copy slice and avoid modifying dimensions.Tags()
+	tb.Append(dimensions.Tags()...)
+	tagger.EnrichTags(tb, dimensions.OriginID(), dimensions.K8sOriginID(), c.cardinality)
+	return tb.Get()
```

Comment:
Since `originID` is always empty, any reason why you don't use `tagger.Tag` ? ([see here](https://github.com/DataDog/datadog-agent/blob/main/pkg/tagger/global.go#L98))
`tagger.Tag` is the standard tagger function and used everywhere (AD, trace agent, process agent etc..). AFAIK `tagger.EnrichTags` is only used by dsd because dsd can provide two origins.

Reply:
Will this step be done if I call `tagger.Tag` though? https://github.com/DataDog/datadog-agent/blob/0cac9362f3426afbc7e370cc3a35ed47730bc1c8/pkg/tagger/global.go#L263-L268

Reply:
You can call [OrchestratorScopeTag()](https://github.com/DataDog/datadog-agent/blob/0cac9362f3426afbc7e370cc3a35ed47730bc1c8/pkg/tagger/global.go#L179) directly if you want